src/constants/inventory.js

```diff
@@ -329,3 +329,65 @@ export let inventory = [
 ];
 
 
+
+// Opdracht 1.a //
+
+const tvTypes = inventory.map((tv) => {
+    return tv.type;
+});
+
+console.log(tvTypes);
+
+
+// Opdracht 1.b //
+
+const tvSoldOut = inventory.filter((tv) => {
+    return  tv.sold === tv.originalStock;
+})
+
+console.log(tvSoldOut);
+
+
+// Opdracht 1.c //
+
+const tvSpecific = inventory.find((tv) => {
+    return tv.type === 'NH3216SMART';
+})
+
+console.log(tvSpecific);
+
+
+// Opdracht 1.d //
+
+const tvSuitableForSport = inventory.map((tv) => {
+    if (tv.refreshRate >= 100) {
+        return 'name: ' + tv.brand + ' ' + tv.name + ', ' + 'suitable: true';
```

Comment:
Ik vind het volgende altijd wat beter te lezen
```suggestion
        return `name: ${tv.brand} ${tv.name}, suitable: true`;
```